tests/behat/theme_boost_union_block_region.feature

```diff
@@ -0,0 +1,553 @@
+@theme @theme_boost_union  @theme_boost_union_block_config @javascript
+Feature: Theme boost union block config
+  In order to add more functionality to pages
+  I need to add blocks to pages
+
+  Background:
+    Given the following "users" exist:
+      | username | firstname | lastname | email |
+      | student1 | Student | 1 | student1@example.com |
+      | student2 | Student | 2 | student2@example.com |
+      | teacher  | teacher | name | teacher@example.com |
+    And the following "courses" exist:
+      | fullname | shortname | format |
+      | Course 1 | C1 | topics |
+    And the following "activities" exist:
```

Comment:
As these tests seem not to do anything with activities, you don't need to create them in the background